src/components/leaderboard/LeaderboardRow/style.module.scss

```diff
@@ -0,0 +1,69 @@
+@use 'src/styles/vars.scss' as vars;
+
+.row {
+  align-items: center;
+  background-color: var(--theme-surface-1);
+  display: grid;
+  font-size: 15px;
+  gap: 32px;
+  grid-template-columns: auto auto 2fr 1fr;
+  height: 64px;
+  padding: 0 32px;
+
+  &[data-style='odd'] {
+    background-color: var(--theme-background);
+  }
+}
+
+.position {
```

Comment:
Let's make sure our elements are nested in a matching order to the HTML code.

E.g. HTML Code
```JSX
<div class="row">
    <span class="position" />
    <img class="picture" />
    <div class="nameRank" >
        <span class="name" />
        <span class="rank" />
    </div>
</div>
```

SCSS Code
```SCSS
.row {
    .position {}
    .picture {}
    .nameRank {
        .name {}
        .rank {}
    }
}
```

One of the biggest benefits I've seen from using this properly is that we can easily refactor code into smaller components by dragging that subset of CSS code out at once. It's also nice for code folding in the editor and helping you debug the element or psuedo-state you're selecting on hover with the right extension and it guarantees you don't make a typo anywhere in the structure of the selector so you don't accidentally select the wrong element.

![Screenshot 2023-06-02 at 2 14 17 PM](https://github.com/acmucsd/membership-portal-ui-v2/assets/33165426/a1de468e-dade-4a90-aad5-3117a717a669)